src/isPasswordActual.test.js

```
@@ -14,15 +14,33 @@ describe(`Function 'isPasswordActual':`, () => {
   });
 
   it(`should return a string`, () => {
+    const result = isPasswordActual(2020, 6, 9);
 
+    expect(result).toEqual(expect.any(String));
   });
 
   it(`should ask to change the password if was changed a year ago`, () => {
-    const lastYear = isPasswordActual(today.year - 1, today.month, today.date);
+    const lastYear = isPasswordActual(2024, today.month, today.date);
 
     expect(lastYear)
       .toBe('Immediately change the password!');
   });
 
-  // write more tests here
+  it(`should approve if password is actual`, () => {
+    const currentYear = isPasswordActual(today.year, today.month, today.date);
+
+    expect(currentYear)
+      .toBe('Password is actual.');
+  });
+
+  it(`should remind if more than 30 days have passed`, () => {
+    const lastMonth = isPasswordActual(
+      today.year,
+      6,
+      5
+    );
+
+    expect(lastMonth)
+      .toBe('You should change your password.');
+  });
```

Issue: The test for 'should remind if more than 30 days have passed' uses hardcoded values (month 6, day 5), which may not always be more than 30 days ago from today, depending on the current date. This can cause the test to fail unpredictably. Please update this test to dynamically calculate a date that is always more than 30 days ago (for example, by creating a new Date object, subtracting 31 days, and extracting the year, month, and date).